HACKING.md

```diff
@@ -21,6 +21,8 @@ To start a ephemeral virtual machine using this repository, run:
 
 `IMAGE` has to be a path to an cloud-init-enabled image matching the host
 operating system, because that's what the packages where built for above.
+Note that the Fedora/RHEL cloud images might be too small for some tests
+to pass. Run `qemu-img resize IMAGE 10G` to grow them.
```

Comment:
Good point. Could you also add a note that the filesystems are grown automatically, thanks to cloud-init's [growpart module](https://cloudinit.readthedocs.io/en/latest/topics/modules.html#growpart), which is enabled by default on the root fs.

Reply:
Like this?